detection-rules/urlhaus_malicious_dom_in_body_or_pdf.yml

```diff
@@ -9,42 +9,45 @@ source: |
         // filter potentially known good domains
         // prevents FPs on entries such as drive[.]google[.]com, or
         // other accidental pushes to the List
-        any([.href_url.domain.domain, .href_url.domain.root_domain], . not in $free_file_hosts)
+      
+        .href_url.domain.domain not in $free_file_hosts
+        and .href_url.domain.root_domain not in $free_file_hosts
         and .href_url.domain.domain not in $tranco_1m
         and .href_url.domain.domain not in $umbrella_1m
-        and 
-
+  
         // this ensures we don't flag on legit FQDNs that
         // aren't in the Tranco 1M, but their root domains are
         // eg: support[.]google[.]com
-        .href_url.domain.root_domain not in $tranco_1m
+        and .href_url.domain.root_domain not in $tranco_1m
         and .href_url.domain.root_domain not in $umbrella_1m
         and .href_url.domain.root_domain not in $free_subdomain_hosts
         and .href_url.domain.root_domain in $abuse_ch_urlhaus_domains_trusted_reporters
     )
     or any(attachments,
-           .file_type == "pdf"
-           and any(file.explode(.),
-                   any(.scan.pdf.urls,
-                       // filter potentially known good domains
-                       // prevents FPs on entries such as drive[.]google[.]com, or
-                       // other accidental pushes to the List
-                       .domain.domain not in $free_file_hosts
-                       and .domain.domain not in $free_subdomain_hosts
-                       and .domain.domain not in $tranco_1m
-                       and .domain.domain not in $umbrella_1m
-                       and 
-
-                       // this ensures we don't flag on legit FQDNs that
-                       // aren't in the Tranco 1M, but their root domains are
-                       // eg: support[.]google[.]com
-                       .domain.root_domain not in $tranco_1m
-                       and .domain.root_domain not in $umbrella_1m
-                       and .domain.domain in $abuse_ch_urlhaus_domains_trusted_reporters
-                   )
-           )
+          .file_type == "pdf"
+          and any(file.explode(.),
+                  any(.scan.pdf.urls,
+                      // filter potentially known good domains
+                      // prevents FPs on entries such as drive[.]google[.]com, or
+                      // other accidental pushes to the List
+                      .domain.domain not in $free_file_hosts
+                      and.domain.root_domain not in $free_file_hosts
```

Comment:
```suggestion
                      and .domain.root_domain not in $free_file_hosts
```

Reply:
You the man. where were you on that one CI checks?